app/controllers/albums_delete_controller.rb

```diff
@@ -0,0 +1,6 @@
+class AlbumsDeleteController < ApplicationController
+  def destroy
+    album = Album.find(params[:id])
+    album.destroy  
+  end
+end
```

Comment:
if album.destroy
   render json: { message: "Delete successful" }
else
   render json : { message: "Delete not successful" }
end